README.md

```diff
@@ -5,7 +5,7 @@
 ## What
 
 `lldb-eval` is an [LLDB](https://lldb.llvm.org/)-based library for evaluating
-debug expressions in the context of the debuggee process. All modern debuggers
+debug expressions in the context of the debugger process. All modern debuggers
```

Comment:
"debuggee" is correct here, the expressions are evaluated in the context of the "process that is being debugged". Let's just use "debugged", if debuggee looks confusing :)

```suggestion
debug expressions in the context of the debugged process. All modern debuggers
```